### ISSUE TITLE
run allcountjs init doesn't work 

### ISSUE THREAD

**Issue by timotanhong** [allcount/allcountjs#112]
root@d4fbd7018d1a:/test# allcountjs init
Usage: allcountjs --app <Path to application's dir or Git URL> --db <Application MongoDB URL> -port [Application HTTP port]

**Comment by matteocng** [comment]
Hi @timotanhong , the output of the following commands could help troubleshooting:

```
lsb_release -sd || cat /etc/*release
allcountjs -V
pgrep mongo
cat app-config/main.js
npm list -g | grep allcountjs
node -v
npm -v
```

**Comment by paveltiunov** [comment]
@timotanhong could you please try to install allcountjs-cli not from root user?

**Comment by timotanhong** [comment]
@matteocng  @paveltiunov  I figure out why, 
I install the another library using npm after I install allcountjs-cli. 
npm install allcountjs 

both of these two packages have allcount.js so the later one 's softlink overwrite the previous one.

**Comment by timotanhong** [comment]
May be the later one need to rename the soft link ?

**Comment by paveltiunov** [comment]
@timotanhong Yep. I see where's the problem. AllcountJS core package is not intended to be installed globally. It's where problem occurs. You're right: seems like we need to rename it.
